manageiq-operator/deploy/crds/manageiq_v1alpha1_manageiq_crd.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: manageiqs.manageiq.example.com
+spec:
+  group: manageiq.example.com
```

Comment:
@Fryguy do we want this to be `manageiq.manageiq.org` as we actually own that domain?

Reply:
can it just be `manageiq.org` without the manageiq subdomain?

Reply:
Yeah, it's just the group so I think that would be fine. The name would be `manageiqs.manageiq.org` then I guess?